client/a8c-for-agencies/sections/sites/sites-dashboard/style.scss

```diff
@@ -461,4 +461,13 @@
 			}
 		}
 	}
+
+	.backup__page {
+		.daily-backup-status__restore-button,
+		.status-card__no-backup-last-backup,
+		.activity-card__toolbar--reverse,
+		.backup-date-picker {
+			display: none;
+		}
+	}
```

Comment:
``` suggestion
```

Reply:
Forget about this suggestion. Let's merge it as it is. I'll add a note to the GH issue about this.